src/pybind/mgr/snap_schedule/fs/schedule_client.py

```diff
@@ -361,7 +361,7 @@ def prune_snapshots(self, sched: Schedule) -> None:
             path = sched.path
             prune_candidates = set()
             time = datetime.now(timezone.utc)
-            mds_max_snaps_per_dir = self.mgr.get_ceph_option('mds_max_snaps_per_dir')
+            mds_max_snaps_per_dir = self.mgr.get_foreign_ceph_option('mds', 'mds_max_snaps_per_dir')
```

Comment:
Could you explain the difference b/w the two calls?

Reply:
`mgr.get_ceph_option()` assumes that the option being requested is a local option i.e. it is a mgr option
`mgr.get_foreign_ceph_option()` does not assume so; it also expects a `who` to get the option from as the first parameter

fyi - I'm still working on the qa commit since `qa/tasks/cephfs/test_snap_schedules.py` is failing at unexpected location